gitconfig.txt

```diff
@@ -1,25 +1 @@
-[user]
-	name = Andres Kievsky
-	email = akievsky@odecee.com.au
-[core]
-	excludesfile = /Users/akievsky/.gitignore_global
-[difftool "sourcetree"]
-	cmd = opendiff \"$LOCAL\" \"$REMOTE\"
-	path = 
-[mergetool "sourcetree"]
-	cmd = /Applications/SourceTree.app/Contents/Resources/opendiff-w.sh \"$LOCAL\" \"$REMOTE\" -ancestor \"$BASE\" -merge \"$MERGED\"
-	trustExitCode = true
-[alias]
-    l = log --graph --abbrev-commit --decorate --format=format:'%C(bold blue)%h%C(reset) - %C(bold green)(%ar)%C(reset) %C(white)%s%C(reset) %C(dim white)- %an%C(reset)%C(bold yellow)%d%C(reset)' --all
-    p = pull --ff-only
-    s = status
-    ss = status -s
-    f = fetch --prune
-    co = checkout
-    d = diff
-    dc = diff --cached
-    bl = branch --list
-    sl = stash list
-    unstage = reset HEAD --
-[rebase]
-	autosquash = true
+# MEXICANS ARE TAKING OUR JOBS
```

Comment:
AM I A PHONY OR IS IT ALL A GIANT CONSPIRACY?! WHAT DOES ALEX JONES THINK?! https://www.youtube.com/watch?v=D3JFB_NfJEM